vscode-antimony/src/server/stibium/stibium/ant_types.py

```diff
@@ -264,7 +264,7 @@ def get_stoich(self):
         return self.children[0].get_value()
 
     def is_const(self):
-        return bool(self.children[1])
+        return self.children[1].text

```

Comment:
not sure why you're changing this here. is_const should always return a boolean value, you could create a new function for getting the text